### ISSUE TITLE
[BUG] - Quasar is not tree-shaked on Nuxt 3.4.2

### ISSUE THREAD

**Issue by Saeid-Za** [Maiquu/nuxt-quasar#48]
### Checks

- [X] I have searched the existing issues for a similar bug and it does not exist!

### Environment

- Operating System: Windows_NT
- Node Version:     v18.14.2
- Nuxt Version:     3.4.2
- Nitro Version:    2.3.3
- Package Manager:  pnpm@7.29.1
- Builder:          vite
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -

### Quasar version

2.11.9

## Description

Hello. Thank you again for this great library.
I've found out that `nuxt-quasar-ui` outputs the whole quasar ui ecosystem when nuxt version is `3.4.2` or higher, meaning that every component source code is present in the final bundle file (`entry.<hash>.js`).
### Nuxt 3.4.2
Here's the vite output emitted from `playground` build logs, notice the `entry` file size. (nuxt `3.4.2`)

<img width="759" alt="image" src="https://user-images.githubusercontent.com/65568529/233836720-5a228930-2ed0-4c10-bfd7-c65c1a4a34cd.png">

### Nuxt 3.4.1 (or lower)
And here's the vite output emitted from `playground` build logs. (nuxt `3.4.1`)

<img width="594" alt="image" src="https://user-images.githubusercontent.com/65568529/233836671-cc80f2c4-91cf-48b2-a983-34fdf5ca5916.png">

## Problem
There was somthing that were changed in nuxt from `3.4.1` -> `3.4.2` that caused this. perhaps it's worthed to check the [changelog](https://github.com/nuxt/nuxt/releases/tag/v3.4.2).

## Reproduction steps

```bash
1. Clone `nuxt-quasar` repo
2. Update Nuxt to `3.4.2`
3. Execute `npm dev:build`
4. Check `entry` file size.
5. Downgrade Nuxt to `3.4.1`
6. Execute `npm dev:build`
6. Check `entry` file size (should be about ~350KB smaller in size)
```

**Comment by Maiquu** [comment]
Culprit: https://github.com/nuxt/nuxt/commit/53db8125f1fed180698510ad132c4c82f8651f5a

The line: 
`import { lazyGlobalComponents } from '#components'` eliminates tree-shaking entirely.

I will look more into it.

**Comment by Maiquu** [comment]
Closed by #51.
